### DISCUSSION TITLE
File Storage

### DISCUSSION THREAD

**Discussion by esilbe1** [vincode-io/Zavala#152] [Support]
Where on the mac are Zavla files stored? I cannot find where to or how to delete them?

**Comment by vincode-io** [comment]
Zavala doesn't store individual outlines in a single file. It uses a combination of files not intended to edited by users. It is next to impossible to determine via the filesystem which files belong to which outlines. If you want to delete them you should use the Zavala user interface.

That said, if you want to delete all the files associated with Zavala you can delete the files in `~/Library/Group Containers/group.io.vincode.Zavala`.

**Comment by esilbe1** [comment]
Thank you. I guess I don’t see how to delete files via the interface.

**Comment by vincode-io** [comment]
You use the context menu in the Documents View (the middle panel).
